articles/uploading-images-to-cloudinary-from-django-application/index.md

```diff
@@ -0,0 +1,270 @@
+Sometimes while working on applications that require the use of images, a simple process like uploading images to a server can become difficult.
+If you deploy your Django to platform like heroku, you can't save image.
+
+In this tutorial, we look at how to save images in Django using CLoudinary.
+
+Cloudinary is an end-to-end image- and video-management solution for websites and mobile apps, covering everything from image and video uploads, storage, manipulations, optimizations to delivery.
+
+### Prerequisites
+
+To follow along with this tutorial, you need Python3 installed on your machine.
+
+### Creating and setting up a new Django project
+
+Let's start by creating a new virtual environment
+
+```bash
+$ python -m virtualenv env
+
+```
+
+Activate virtual environment
+
+```bash
+$ source virtual/bin/activate
+
+```
+
+Install Django
+
+```bash
+$ pip install django
+
+```
+
+Create new Django project
+
+```bash
+$ django-admin startproject photoapp
+
+```
+
+Create Django app
+
+```bash
+$ django-admin startapp photos
+
+```
+
+Lets add our app to the list of installed apps. Navigate to the "photoapp" directory and edit the settings.py file
+
+```python
+INSTALLED_APPS = [
+
+    'django.contrib.admin',
+    
+    'django.contrib.auth',
+    
+    'django.contrib.contenttypes',
+    
+    'django.contrib.sessions',
+    
+    'django.contrib.messages',
+    
+    'django.contrib.staticfiles',
+    
+    'photos',
+    ]
+```
+
+### Setting up Cloudinary
+
+Now lets head over to [cloudinary](https://cloudinary.com/) website to create an account.
+Click on the Sign Up button and fill in your details to create an account.
+
+### Installation
+
+Install the cloudinary module in our project
+
+```bash
+$ pip install cloudinary
+
+```
+
+Next thing is to add Cloudinary to the list of installed apps in settings.py
+
+
+```python
+INSTALLED_APPS = [
+
+    'django.contrib.admin',
+    
+    'django.contrib.auth',
+    
+    'django.contrib.contenttypes',
+    
+    'django.contrib.sessions',
+    
+    'django.contrib.messages',
+    
+    'django.contrib.staticfiles',
+    
+    'photos',
+    'cloudinary'
+    ]
+```
+
+We will also need to include Cloudinary's python classes in settings.py
+
+```python
+    import cloudinary
+    import cloudinary.uploader
+    import cloudinary.api
+```
+
+### Configuration
+
+To use the Cloudinary Django library, we have to configure at least our cloud_name, api_key and api_secret .
+we can find our account-specific configuration credentials in the Dashboard page of the account console.
+
+![Cloudinary Dashboard](/engineering-education/uploading-images-to-cloudinary-from-django-application/dash.png)
```

Comment:
Add image source (if not yours), and add an image footer describing about the image. Something like this:

*Cloudinary dashboard page*